### PR TITLE
docs: Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by zyxiaoyuer**

### CODE REVIEW

File: packages/vscode-extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+
+To enable the preview features, refer to the [preview guidance](https://github.com/OfficeDev/TeamsFx/wiki/Enable-Preview-Features-in-Teams-Toolkit)
+
+Enhancement:
+
+- Improves UI experience and command pallette experience.
+- UX A/B testing:
+  - Your Tree View(sidebar) may include or exclue quick start page.
+  - You may or may not be invited to do local debug after create new project.
+- SDK add certificate authentication.
```
Comment (formulahendry) [inline packages/vscode-extension/CHANGELOG.md:25]: Add certificate-based authentication for JavaScript SDK

File: packages/vscode-extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,29 @@
 # Changelog
 
+
+## 2.7.1 - Oct 18, 2021
+
+Incremental version for Teams Toolkit with multiple bugs fixes and the following updates:
+
+New Feature:
+- Support Jenkins CI/CD pipeline.
+- Support certificate-based authentication for JavaScript SDK
```
Comment (eriolchan) [inline packages/vscode-extension/CHANGELOG.md:10]: Do we need to mention what's the SDK API and what's the additional parameter for this certificate support?

File: packages/vscode-extension/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,29 @@
...
+
+Preview Features:
+- Support management of multiple development environments.
+- Support project collaborations among multiple developers.
+- Support customize Azure resource provision.
+- support customize manifest.
+
+To enable the preview features, refer to the [preview guidance](https://github.com/OfficeDev/TeamsFx/wiki/Enable-Preview-Features-in-Teams-Toolkit)
```
[the remark this answers is not in the document]
Comment (zyxiaoyuer) [reply packages/vscode-extension/CHANGELOG.md:14]: we should added it to wiki page

File: packages/vscode-extension/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+
+## 2.7.1 - Oct 18, 2021
```
Comment (eriolchan) [inline packages/vscode-extension/CHANGELOG.md:4]: Why don't we use v2.8.0?

File: packages/vscode-extension/WHATISNEW.md [hunk 5]
```diff
@@ -1,4 +1,29 @@
...
+- Support customize manifest.
+- Improve the experience to customize Azure resource provision using ARM(Azure Resource Manager).
+
+The picture below shows how to enable preview features, more details refer to the [preview guidance](https://github.com/OfficeDev/TeamsFx/wiki/Enable-Preview-Features-in-Teams-Toolkit)
+
+![Alt Text](https://raw.githubusercontent.com/OfficeDev/TeamsFx/main/packages/vscode-extension/img/enable-preview.gif)
+
+Enhancement:
+
+- Improve UI experience and Command Pallette experience. Ensure the same experience between Tree View and Command pallette.
```
Comment (zhenjiao-ms) [inline packages/vscode-extension/WHATISNEW.md:21]: It is not clear of what the change is? Is the enhancement for "making the command and tree view text be consistent"?

File: packages/vscode-extension/WHATISNEW.md [hunk 6]
```diff
@@ -1,4 +1,29 @@
...
+- UX A/B testing:
+  - Your Tree View(sidebar) may include or exclude quick start page.
```
Comment (zhenjiao-ms) [inline packages/vscode-extension/WHATISNEW.md:23]: It is better to add the screenshot to better explain it

File: packages/vscode-extension/WHATISNEW.md [hunk 7]
```diff
@@ -1,4 +1,29 @@
...
+
+- [2.8.0 - Oct 18 2021](#280---oct-18-2021)
+- [2.7.0 - Sep 17 2021](#270---sep-17-2021)
+
+## 2.8.0 - Oct 18 2021
+
+Incremental version for Teams Toolkit with multiple bugs fixes and the following updates:
+
+Preview Features:
+- Support management of multiple environments.
```
Comment (zhenjiao-ms) [inline packages/vscode-extension/WHATISNEW.md:10]: Can we add a simple screenshot for each item so that people will be more active to try them

**Review by zhenjiao-ms — CHANGES_REQUESTED** [review]

File: packages/vscode-extension/CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,25 @@
...
 
+
+## 2.8.0 - Oct 18, 2021
+
+Incremental version for Teams Toolkit with multiple bugs fixes and the following updates:
+
+Preview Features:
+- Support management of multiple environments.
+- Support project collaborations among multiple developers.
+- Support customize manifest.
```
Comment (zhenjiao-ms) [inline packages/vscode-extension/CHANGELOG.md:11]: What is this for?

File: packages/vscode-extension/CHANGELOG.md (hunk 4 above)
Comment (adashen) [reply packages/vscode-extension/CHANGELOG.md]: The rc version is 2.8.0, it is correct.

File: packages/sdk/CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,6 @@
+# 0.2.7
```
Comment (formulahendry) [inline packages/sdk/CHANGELOG.md:1]: Please double check with release manager. The version is 0.3.0.

Comment (zyxiaoyuer) [reply packages/sdk/CHANGELOG.md]: Corrected, thanks

**Pull request merged**